Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-openshift-4.6 AS builder
+WORKDIR /go/src/github.com/openshift/cluster-api-provider-packet
```

Comment:
Who can help us rename this project to `cluster-api-provider-equinix-metal` (or hyphenated `equinixmetal`, I don't know if there is a preference for technical or historical reasons)